nodelets.md

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
We should add a reference to this file in `README.md` or it will likely never get read.

Reply:
:heavy_check_mark: